britnay268/INDIVIDUAL-ASSESSMENT-vocab-YOU-lary
# Stretch goal3

## Description
A user can toggle a card off which means the card shows as public and is in the community section meaning the card is public for anyone to see. If the card is toggle as private, only the user can see it.

## Related Issue
#19 Partially

## Motivation and Context
Wanted the user to have the opportunity to create a card to make it public for other users to see or just for them to see.

## How Can This Be Tested?
Toggle a card to private and see if you see it in the community tab. This would mean that only you can see that card. Toggle the card off and it will show up in community.

## Types of changes
